orbital4c/NuclearPotential.py

```diff
@@ -26,3 +26,75 @@ def uHFYGB(r):
     u = erf(r)/r + (1/(3*np.sqrt(np.pi)))*(np.exp(-(r**2)) + 16*np.exp(-4*r**2))
     return u
 
+def PoCh(position, center , charge):
+    d2 = ((position[0] - center[0])**2 +
+          (position[1] - center[1])**2 +
+          (position[2] - center[2])**2)
+    distance = np.sqrt(d2)
+    return charge / distance
+
+def HomChSph(position, center, charge, atom):
```

Comment:
Use more "meaningful" names. In 1y you will not remember what `HomChSph` means. Use rather `HomogeneousChargePotential`. Your future self will thank you for that :-)